blark/tests/test_transformer.py

```diff
@@ -836,6 +836,15 @@ def test_fb_roundtrip(rule_name, value):
             END_CASE
             """
         )),
+        param("case_statement", tf.multiline_code_block(
+            """
+            CASE expr OF
+            BYTE#9..BYTE#10, BYTE#13, BYTE#28:
+                OneToTen := OneToTen + 1;
+            EnumValue:
+            END_CASE
+            """
+        )),
```

Comment:
Would you mind terribly adding coverage for the boolean case as well? It's not a big deal, but it would be nice 😁

Reply:
oh, of course! I should've thought of that :facepalm: